experiments/calibration/minimal_working_example/minimal_working_example.jl

```diff
@@ -0,0 +1,96 @@
+# This script describes the simplest possible calibration of ClimaLand,
+# with step by step explanations.
+# If one line or comment confuses you, or you believe it could be simplified,
+# reach out to Alexis Renchon.
+
+# We calibrate a single site simulation, using EnsembleKalmanProcesses.jl.
+# We calibrate the parameters g1 and g0, to better match canopy transpiration target data.
+
+# To perform this calibration we will need:
+# 1. a function returning our model transpiration output given the parameters we want to calibrate.
+# 2. the "truth" target data, to calibrate on.
+# 3. the prior distribution of these parameters.
+
+# Let's go!
+
+# Packages
+import ClimaLandSimulations.Fluxnet as CLS # to run the model
+import EnsembleKalmanProcesses as EKP # to perform the calibration
+import Random # we need it for ...
+
+# a function returning our model transpiration output given the parameters to calibrate
+function ClimaLand_transpiration(params) # params is a 2 element Array
+    g1, g0 = params
+
+    sv = CLS.run_fluxnet(
+                         "US-MOz";
+                         params = CLS.ozark_default_params(; conductance = CLS.conductance_ozark(; g1 = g1, g0 = g0)),
+                        )[1];
+
+    inputs = CLS.make_inputs_df("US-MOz")[1];
+
+    simulation_output = CLS.make_output_df("US-MOz", sv, inputs);
+
+    transpiration = simulation_output.transpiration .* 1e9 # 1e9 because very small otherwise
```

Comment:
Why the 1e9? Is there an issue with floating point precision, or is there an error in the output?

Reply:
This also changes the float type